### PR TITLE
chore: tweaks to beam

### PR DESCRIPTION
**Pull request by igorbernstein2**
- remove unnecessary dependency plugin disablement
- force upgrade commons-compress
- move plugin config overrides to pluginManagement

### CODE REVIEW

File: bigtable-dataflow-parent/bigtable-beam-import/pom.xml [hunk 1]
```diff
@@ -509,33 +511,6 @@ limitations under the License.
           </execution>
         </executions>
       </plugin>
-
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-failsafe-plugin</artifactId>
-        <configuration>
-          <!-- disable integration tests by default, opt-in via profile -->
-          <skip>true</skip>
```
Comment (kolea2) [inline bigtable-dataflow-parent/bigtable-beam-import/pom.xml:518]: can you confirm tests are running as expected?

File: bigtable-dataflow-parent/bigtable-beam-import/pom.xml [hunk 2]
```diff
@@ -421,6 +421,8 @@ limitations under the License.
         <filtering>false</filtering>
       </testResource>
     </testResources>
+
```
Comment (kolea2) [inline bigtable-dataflow-parent/bigtable-beam-import/pom.xml:424]: nit - revert

File: bigtable-dataflow-parent/bigtable-beam-import/pom.xml (hunk 1 above)
Comment (igorbernstein2) [reply bigtable-dataflow-parent/bigtable-beam-import/pom.xml:518]: yep

File: bigtable-dataflow-parent/bigtable-beam-import/pom.xml (hunk 2 above)
Comment (igorbernstein2) [reply bigtable-dataflow-parent/bigtable-beam-import/pom.xml:424]: done

**Review by kolea2 — APPROVED** [review]

**Pull request merged**